sdk/python/kfp/dsl/artifact.py

```diff
@@ -184,6 +184,11 @@ def get_artifact_type(cls) -> str:
 
     return serialization_utils.yaml_dump(result_map)
 
+  @classmethod
+  def get_ir_type(cls) -> pipeline_spec_pb2.ArtifactTypeSchema:
```

Comment:
nit: maybe rename to `get_artifact_type_schema` ?

Reply:
Will do it in a follow up PR as well. This file might not be needed once we merge ontology_types into io_types.